distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc

```diff
@@ -0,0 +1,108 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="distributed-tracing-rn-3-1"]
+= Release notes for {DTProductName} 3.1
+:context: distributed-tracing-rn-3-1
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
+
+[id="distributed-tracing-rn_3-1_component-versions"]
+== Component versions in the {DTProductName} 3.1
+
+[options="header"]
+|===
+|Operator |Component |Version
+
+|{TempoName}
+|Tempo
+|2.3.1
+
+//for each new release, update the release number in the xref path on the following line
+|xref:../../otel/otel_rn/otel-rn-3.1.adoc[{OTELName}]
+|OpenTelemetry
+|0.93.0
+
+|{JaegerName} (deprecated)
+|Jaeger
+|1.53.0
+
+|===
```

Comment:
It seems odd that there's a link for one here and not the others - maybe just leave the link out?

Reply:
This was specifically requested by the product manager because previously OpenTelemetry was part of the Distributed Tracing docs but was then moved out of the Distributed Tracing docs to become a separate item in the top-level navigation menu of the OCP docs. AFAIK, this link is to promote OpenTelemetry, but I expect to remove this link from the Distributed Tracing docs for a future release, perhaps even for the next 3.1 release if we get sufficient content in it to have the OpenTelemetry docs separate from the Distributed Tracing docs.